packages/relay/src/lib/eth.ts

```diff
@@ -1210,6 +1210,10 @@ export class EthImpl implements Eth {
         constants.TYPE_TOKEN,
       ]);
       if (result) {
+        const blockInfo = await this.getBlockInfo(blockNumber, requestDetails);
```

Comment:
another thing came to mind, what if the token was/is deleted, but in the `blockNumber` requested the token still exists? In this case, the result of `resolveEntityType` would be falsy, so there would be no block comparison.

If this is an issue, I don't mind tackling this in another PR.

Reply:
Hmm, I just tested with some tokens that were later deleted that I found on testnet (for example - https://hashscan.io/testnet/token/0.0.4933576) and I passed the creation block and it returned the bytecode. The result was not falsy,  it just had `"deleted": true,`

Reply:
ok, so that means if the now the `blockNumber` is after the token was deleted, we still be getting the bytecode when we shoudn't, right?

Reply:
Yes, that is correct.

Reply:
Leaving this unsolved for now since getting token by ID would always return `deleted: true` even for an older date